docs/sources/flow/install/linux.md

```diff
@@ -55,54 +49,68 @@ or RedHat Enterprise Linux), complete the following steps:
    sslcacert=/etc/pki/tls/certs/ca-bundle.crt
    ```
 
-2. Verify that the repository is properly configured using
-   `yum-config-manager`:
+2. Verify that the repository is properly configured using `yum-config-manager`:
 
    ```shell
    yum-config-manager grafana
    ```
 
-3. Install the Grafana Agent Flow package:
+3. Install Grafana Agent Flow:
 
    ```shell
    sudo yum install grafana-agent-flow
    ```
 
+## Install on SUSE or openSUSE
+
+To install Grafana Agent Flow on SUSE or openSUSE, complete the following steps:
+
+1. Open a terminal and run the following to install Grafana’s package repository:
+
+   ```shell
+   wget -q -O gpg.key https://apt.grafana.com/gpg.key
+   sudo rpm --import gpg.key
+   sudo zypper addrepo https://rpm.grafana.com grafana
+   ```
+
+1. Update the repository and install Grafana Agent:
+
+   ```shell
+   sudo zypper update
+   sudo zypper install grafana-agent
```

Comment:
@clayton-cornell Should this be grafana-agent-flow?

Reply:
@rfratto D'oh! Yes it should. Fixed.